samples/dynamodb/simple/DynamoDB_1/Client/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.DynamoDB.Simple.Client";
+        var endpointConfiguration = new EndpointConfiguration("Samples.DynamoDB.Simple.Client");
+        endpointConfiguration.UsePersistence<LearningPersistence>();
```

Comment:
Should this be Dynamo?

Reply:
the client doesn't need any persister, but there's not much value in configuring dynamo as the persister. however we should be able to remove the persistence config completely.

Reply:
removed the persistence config from the client config completely